app/src/main/java/com/github/multimatum_team/multimatum/MultimatumApp.kt

```diff
@@ -0,0 +1,11 @@
+package com.github.multimatum_team.multimatum
+
+import android.app.Application
+import dagger.hilt.android.HiltAndroidApp
+
+@HiltAndroidApp
+class MultimatumApp: Application() {
+
+
+
+}
```

Comment:
What is the purpose of this file?

Reply:
Quoting the documentation:
> All apps that use Hilt must contain an [Application](https://developer.android.com/reference/android/app/Application) class that is annotated with @HiltAndroidApp.

I am not exactly sure why this is required, but it is probably because it is the only place where we can put the `@HiltAndroidApp` annotation that triggers code generation.
But you are right, it is not self-explanatory. I am going to add a comment in the file.